api_server_new.py

```
@@ -0,0 +1,1558 @@
+        # If all_thumbnail_urls exists and is already a list, use it
+        # If it's a JSON string, parse it
+        # If it doesn't exist, create an empty list
+        if 'all_thumbnail_urls' in clip and clip['all_thumbnail_urls']:
+            if isinstance(clip['all_thumbnail_urls'], list):
+                # Already in the right format
+                pass
+            elif isinstance(clip['all_thumbnail_urls'], str):
+                # Try to parse JSON string
+                try:
+                    clip['all_thumbnail_urls'] = json.loads(clip['all_thumbnail_urls'])
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)